backend/package.json

```diff
@@ -9,6 +9,8 @@
     "prebuild": "rimraf dist",
     "build": "nest build",
     "format": "prettier --write \"apps/**/*.ts\" \"libs/**/*.ts\"",
+    "prestart:backend": "rm -rf dist && nest build backend && nest build shared",
```

Comment:
buliding shared should be necessary, `build backend` will grab what it needs from shared